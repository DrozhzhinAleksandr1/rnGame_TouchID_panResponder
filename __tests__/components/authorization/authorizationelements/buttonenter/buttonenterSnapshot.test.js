import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {InputPassWordWithSVG}  from '../../../../../app/components/authorization/authoriztionelements/textinput';
import { localization } from '../../../../../app/common/localization';


it('create snapshot InputPassWordWithSVG', () => {
    const renderer = new ShallowRenderer()
    let buttonRegistrationSnapshot = renderer.render(<InputPassWordWithSVG CurrentLanguage={localization.ru}/>);
    expect(buttonRegistrationSnapshot).toMatchSnapshot();
});