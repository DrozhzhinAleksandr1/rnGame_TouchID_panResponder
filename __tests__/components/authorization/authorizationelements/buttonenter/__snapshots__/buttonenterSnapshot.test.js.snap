// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot InputPassWordWithSVG 1`] = `
<Component
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Component
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "borderColor": "#FFFFFF",
          "borderWidth": 2,
          "flexDirection": "row",
          "height": 40,
          "justifyContent": "space-between",
          "paddingLeft": 15,
          "paddingRight": 15,
          "width": 296,
        },
        null,
      ]
    }
  >
    <TextInput
      accessibilityLabel="PasswordInputAuthorizationID"
      allowFontScaling={true}
      autoCorrect={false}
      onBlur={[Function]}
      onChangeText={[Function]}
      placeholderTextColor="#CECECE"
      secureTextEntry={true}
      selectTextOnFocus={false}
      selectionColor="#5A6772"
      spellCheck={false}
      style={
        Array [
          Object {
            "backgroundColor": "white",
            "color": "#000",
            "fontFamily": "Roboto-Light",
            "fontSize": 18,
            "height": "100%",
            "padding": 0,
            "width": 252,
          },
          null,
        ]
      }
      testID="PasswordInputAuthorizationID"
      underlineColorAndroid="white"
      value=""
    />
    <TouchableWithoutFeedback
      accessibilityLabel="BtnShowPassAuthorizationID"
      onPress={[Function]}
      testID="BtnShowPassAuthorizationID"
    >
      <Component>
        <Dontshow />
      </Component>
    </TouchableWithoutFeedback>
  </Component>
  <ShowWarning
    isshowwarning={false}
    testID="PasswordErrorAuthorizationID"
    text="Ошибка ввода пароля"
  />
</Component>
`;
